lib/src/over_react_redux/over_react_flux.dart

```diff
@@ -309,7 +309,7 @@ mixin InfluxStoreMixin<S> on flux.Store {
   }
 }
 
-bool _shallowMapEquality(Map a, Map b) => const MapEquality().equals(a, b);
+bool _shallowMapIdentityEquality(Map a, Map b) => const MapEquality(keys: IdentityEquality(), values: IdentityEquality()).equals(a, b);
```

Comment:
It's mainly for values (I actually need to update with this a more optimal solution), using `identical` instead of normal equality to compare objects, since normal equality can be expensive.

So, instead of calling `operator==` on the LHS of the object to do comparison, it uses `identical`, which checks whether two objects are the same instance (or for primitives, if they have the same value).

Reply:
Ohh I think I didn't word my question well. I was more of asking specifically around `keys: IdentityEquality(), values: IdentityEquality()` (the parameters to the equality). It doesn't seem like those parameters are needed

Reply:
Ah, by default they use `DefaultEquality`, which uses `operator==`-based equality. I wanted `identical`-based equality, so I specified `IdentityEquality`